src/modules/gdgem/srs/BuilderVMM3.h

```diff
@@ -37,6 +38,9 @@ class BuilderVMM3 : public AbstractBuilder {
   /// \todo Martin document
   void process_buffer(char *buf, size_t size) override;
 
+  /// \brief return the current filename
+  std::string getFilename() { return readout_file_->get_full_path().string(); }
```

Comment:
It sort of does - by crashing. It is a programming error to use getFilename 
in the wrong context. But I have updated the code comment and added 
a runtime exception, so that we end up with the same crashing behaviour.